e2e/test/tracing_and_performance.bats

```diff
@@ -15,7 +15,7 @@ teardown() {
     tsconfig
     ts_project --src "source.ts"
     echo "export const f = 1;" > source.ts
-    run bazel build :foo
+    run bazel build :foo --@aspect_rules_ts//ts:supports_workers
```

Comment:
nit: you don't need these here as it's explicitly turned on in .bazelrc

Reply:
I did need this to make the PR green. It's only in e2e/worker that the bazelrc contains the line, but AFAICT that directory doesn't contribute into these e2e setups under bats.